lib/mockbuilder.py

```diff
@@ -195,12 +203,32 @@ def copy_results(self, package):
             self.timestamp, package.name)
         self._copy_rpms(package.build_cache_dir, package_build_results_dir)
 
+    def create_repository(self):
+        """
+        Create yum repository in build results directory.
+        """
+        result_dir = CONF.get('common').get('result_dir')
+        build_results_dir = os.path.join(
+            result_dir, 'packages', self.timestamp)
+        utils.run_command("createrepo %s" % build_results_dir)
+
+        repo_files_dir = os.path.join(result_dir, "repository_files")
```

Comment:
I don't like bike-shedding discussion but `repository_files` sounds awkward to me.

What if we shortened it to just `repository` then `result` directory would look like the following:

```
results/repository/
results/iso/
...
```

It's implicit that some *files* will be inside *repository* directory, we don't need to re-state that in the directory name.

We can still change it a little bit to something similar to what is done in [CentOS image](http://mirror.centos.org/altarch/7/os/ppc64le/):

```
result/packages/
result/repodata/
result/iso/
...
```

What do you think?